src/DynamoCoreWpf/Views/PackageManager/InstalledPackagesView.xaml

```diff
@@ -1,255 +1,342 @@
-        Title="{x:Static p:Resources.InstalledPackageViewTitle}" 
-        Height="450" Width="550" Background="#555" MinWidth="550" MaxWidth="550" 
+        Title="{x:Static p:Resources.InstalledPackageViewTitle}"
+        Height="450"
+        Width="550"
+        Background="#555"
+        MinWidth="550"
+        MaxWidth="550"
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
         xmlns:ViewModels="clr-namespace:Dynamo.ViewModels"
         mc:Ignorable="d"
         d:DataContext="{d:DesignInstance ViewModels:InstalledPackagesViewModel, IsDesignTimeCreatable=False}">
 
-        <Window.Resources>
-            <ResourceDictionary>
-                <ResourceDictionary.MergedDictionaries>
+    <Window.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
                 <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoModernDictionaryUri}" />
-                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoConvertersDictionaryUri}"/>
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoConvertersDictionaryUri}" />
                 <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoColorsAndBrushesDictionaryUri}" />
-                </ResourceDictionary.MergedDictionaries>
-            </ResourceDictionary>
-        </Window.Resources>
-        
-    <Grid Background="Black" Margin="0">
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Window.Resources>
+
+    <Grid Background="Black"
+          Margin="0">
 
         <Grid.RowDefinitions>
-            <RowDefinition Height="Auto"/>
-            <RowDefinition Height="*"/>
-            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto" />
         </Grid.RowDefinitions>
-        
-        <ScrollViewer HorizontalAlignment="Stretch" Margin="0" Name="ScrollView" VerticalAlignment="Stretch" Focusable="False"
-                      VerticalScrollBarVisibility="Visible" Background="Black" BorderThickness="0" Grid.Column="0" Grid.ColumnSpan="1" Grid.Row="1" >
-
-            <ListBox Name="SearchResultsListBox" ItemsSource="{Binding Path=LocalPackages}" BorderThickness="0" Padding="0" Background="Black" >
-
-                    <ListBox.ItemContainerStyle>
-                        <Style TargetType="ListBoxItem">
-                            <Style.Resources>
-                                <SolidColorBrush x:Key="{x:Static SystemColors.HighlightBrushKey}" Color="#000"/>
-                                <SolidColorBrush x:Key="{x:Static SystemColors.ControlBrushKey}" Color="#000" />
-                            </Style.Resources>
-                        </Style>
-
-                    </ListBox.ItemContainerStyle>
```

Comment:
The `ScrollViewer` that encapsulates the `ListBox` has been removed since `ListBox` itself has an internal `ScrollViewer`. The scrolling failed because the `ListBox` was allowed to grow inside `ScrollViewer` for as long as it needs to be, but it doesn't show a vertical scrollbar (which is shown by the `ScrollViewer` that contains it). When mouse wheel messages get to the `ListBox`, they get handled but since `ListBox` is tall enough to accommodate its content, scrolling did not take place and the messages consumed.